### ISSUE TITLE
Port existing journal workflows to Holding Pen

### ISSUE THREAD

**Issue by jalavik** [inspirehep/inspire-next#269]
The existing journal workflows need to be ported to use

- [ ] Unify harvesting workflow into one workflow for all sources by modifying `arxiv_harvest_math` to be generic

Add specific workflows/tasks for each source:
- [ ] World Scientific
- [ ] Elsevier
- [ ] APS
- [ ] IOP
- [ ] PoS

FIXME: expand this issue with more info

**Comment by kaplun** [comment]
@jalavik Can you provide a status update of this task?

**Comment by fschwenn** [comment]
Stupid question: APS is ticked - does it mean it is considered 'completed'? I somehow can not see how it handles references?

**Comment by jalavik** [comment]
@fschwenn You are right. Not all of them are 100% completed yet, but at least it exists and produces some metadata and has an integration with the source, which is why I ticked it off. Many of the crawlers probably need more love to be 100% good. We should add missing features as issues on the hepcrawl github issues. 

In regards to references, I see now that their new JSON api does not seem to include them http://harvest.aps.org/docs/harvest-api#general so we need to request and parse the xml format additionally for this info:

```
curl  -H 'Accept: text/xml' http://harvest.aps.org/v2/journals/articles/10.1103/PhysRevSTAB.4.072801
```

**Comment by kaplun** [comment]
@inspirehep/inspire-dir Maybe asking to publishers to provide references through their API could be a topic for the next AAHEP?

**Comment by kaplun** [comment]
@bittirousku @david-caro @fschwenn  do you know what is the status of Springer and EDPSciences ++? Are there open PRs that can be linked to this issue?

**Comment by ksachs** [comment]
Florian is working on the Springer Crawler. We get references in the feeds. It's mainly IOP that doesn't provide references.

**Comment by david-caro** [comment]
We have inspirehep/hepcrawl#43 for the EDP one, I'm not sure about the springer one.
